src/main/kotlin/app/revanced/patches/youtube/general/snackbar/ForceSnackbarTheme.kt

```diff
@@ -0,0 +1,130 @@
+package app.revanced.patches.youtube.general.snackbar
+
+import app.revanced.patcher.data.ResourceContext
+import app.revanced.patcher.patch.options.PatchOption.PatchExtensions.stringPatchOption
+import app.revanced.patches.youtube.utils.compatibility.Constants.COMPATIBLE_PACKAGE
+import app.revanced.patches.youtube.utils.settings.SettingsPatch
+import app.revanced.util.doRecursively
+import app.revanced.util.insertNode
+import app.revanced.util.patch.BaseResourcePatch
+import org.w3c.dom.Element
+
+@Suppress("DEPRECATION", "unused")
+object ForceSnackbarTheme : BaseResourcePatch(
```

Comment:
Since some snackbar themes do not seem to change, this patch will not be reflected in PR
(Theme of the snackbar should be changed programmatically)

If I find a way to change the theme of all snackbars in the future or find a way to apply the theme of snackbar programmatically, I will add it to the patch